### ISSUE TITLE
No exe file

### ISSUE THREAD

**Issue by TurboS1ayer** [ItzBlinkzy/roblox-rpc#7]
I'm probably just being stupid, but as the title says, but where is the exe file??

**Comment by TurboS1ayer** [comment]
I was indeed being stupid. I found the setup and ran it, BUT IT DOESN'T WORK. I've met every prerequisite, but absolutely nothing. It refuses to show my status. Also, nothing happens when I right click the application in my system tray.

**Comment by ItzBlinkzy** [comment]
Hey, the current version of ROBLOX RPC is really buggy and doesn't work half the time. As I'm typing this I am in the process of a major haul for V2 of ROBLOX RPC, it should be ready in 1-2 weeks or possibly next week.
I will come back here and let you know once it is ready.

(A little preview)
![image](https://github.com/ItzBlinkzy/roblox-rpc/assets/68260779/f89bc6f3-7a5c-47c6-881f-59138d9704ca)

**Comment by ItzBlinkzy** [comment]
I've released [ROBLOX RPC V2](https://github.com/ItzBlinkzy/roblox-rpc/releases/tag/v2.0.0) earlier than I anticipated. It should be working now and with some better QOL improvements.

If you come across any issues with v2 don't hesitate to create another issue or ask in the [Discord Server](https://discord.com/invite/aq9rwUCQrK).
